storage/innobase/xtrabackup/doc/source/installation/compiling_xtrabackup.rst

```diff
@@ -13,6 +13,7 @@ branch that you want to install, such as **8.0**.
 
   $ git clone https://github.com/percona/percona-xtrabackup.git
   $ cd percona-xtrabackup
+  $ git submodule update --init --recursive
   $ git checkout 8.0
```

Comment:
maybe we should first checkout the 8.0 branch then initialize the submodules?

Reply:
Ah true!